services/donationService.ts

```diff
@@ -69,3 +72,19 @@ const notifyTransakUpdate = async donation => {
     }
   }
 };
+
+export const updateTotalDonationsOfProject = async (projectId: number) => {
+  try {
+    const donationsAmount = await Donation.query(
+      `select COALESCE(SUM("valueUsd"),0) as total from donation where "projectId" = ${projectId}`,
```

Comment:
This is just conventions in sql, writing the commands capitalized and separated per line.
```javascript
SELECT COALESCE(SUM("valueUsd"),0) AS total
FROM donation
WHERE "projectId" = ${projectId}
```

Reply:
Thanks Carlos, , I wasn't aware about conventions, now I changed it